soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -5069,7 +5052,9 @@ void func_8083A0F4(PlayState* play, Player* this) {
             this->interactRangeActor->parent = &this->actor;
             Player_SetupAction(play, this, Player_Action_8084F608, 0);
             this->stateFlags1 |= PLAYER_STATE1_IN_CUTSCENE;
-            sMaskMemory = PLAYER_MASK_NONE;
+            if (!CVarGetInteger(CVAR_ENHANCEMENT("PersistentMasks"), 0) || !CVarGetInteger(CVAR_ENHANCEMENT("AdultMasks"), 0)) {
```

Comment:
Regarding efficiency, it's the same as the compiler optimizes it.
Regarding it being clearer, i disagree, I'm not sure what you want me to do, invert the condition, have an empty if and add the `gSaveContext.maskMemory = PLAYER_MASK_NONE;` inside an else?

Reply:
My idea was just to invert the nor condition into an and. if I misread something and it would be messy you can skip this, I'm trying to get used to checking other people's code quality.